src/common/modules/data/DefaultSettings.js

```diff
@@ -16,7 +19,7 @@ const defaultSettings = Object.freeze({
     popupIconColored: false,
     qrCodeType: "svg",
     qrColor: "#0c0c0d",
-    qrBackgroundColor: "#ffffff",
+    qrBackgroundColor: (darkTheme ? "#d7d7db" : "#ffffff"),
```

Comment:
GitHub makes funny diffs, BTW:
![image](https://user-images.githubusercontent.com/11966684/58984691-ee77ea80-87d9-11e9-9869-1cb452626323.png)

Anyway, to get serious, you missed to document that "magic value"=color there now…

Reply:
```suggestion
    qrBackgroundColor: (darkTheme ? "#d7d7db" : "#ffffff"), // dark uses Firefox Photon's grey-30
```

Reply:
```suggestion
    qrBackgroundColor: (isDarkTheme ? "#d7d7db" : "#ffffff"),
```

also obviously

Reply:
pushed this up too

Reply:
Thanks yeah, just clicking "resolve conversation" should do it. :smile: